interface/forms/vitals/templates/vitals/general_new.html

```diff
@@ -145,230 +145,227 @@
-                            {/foreach}</tr>
-
-                            {if $units_of_measurement == 3}<tr class="hide">{else}<tr>{/if}
-                                        {if $units_of_measurement == 1}<td class="unfocus graph" id="weight_metric">{else}<td class="graph" id="weight_metric">{/if}{xlt t="Weight"}</td>
-                                {if $units_of_measurement == 1}<td class="unfocus">{else}<td>{/if}{xlt t="kg"}</td>
-                                        {if $units_of_measurement == 1}<td class="valuesunfocus">{else}<td class='currentvalues'>{/if}
-                                        <input type="text" class="form-control" size='5' id='weight_input_metric' value="{if $vitals->get_weight() != 0}{math equation="number * constant" number=$vitals->get_weight() constant=0.45359237 format="%.2f"|attr}{/if}" onChange="convKgtoLb('weight_input');"/>
-                                        </td>
+    <div class="row">
+        <h2>{xlt t="Vitals"}</h2>
+        <div class="col-sm-12">
+            <form name="vitals" method="post" action="{$FORM_ACTION}/interface/forms/vitals/save.php" onSubmit="return vitalsFormSubmitted()">
+                <input type="hidden" name="csrf_token_form" value="{$CSRF_TOKEN_FORM|attr}" />
+                <div id="chart" class="chart-dygraphs" style="margin-left: -15px"></div>
+                    <div class="table-responsive">
+                        <table class="table table-sm">
+                            <thead>
+                                <tr>
+                                    <th class="text-left">{xlt t="Name"}</th>
+                                    <th class="text-left">{xlt t="Unit"}</th>
+                                    <th class='currentvalues' title="{xla t='Date and time of this observation'}">
+                                        <input type='text' size='14' class='form-control datetimepicker oe-patient-background' name='date' id='date' value='{$vitals->get_date()|date_format:"%Y-%m-%d %H:%M"|attr}' />
+                                    </th>
+                                    {foreach item=result from=$results}
+                                        <th class='historicalvalues'>{$result.date|date_format:"%Y-%m-%d %H:%M"|text}</th>
+                                    {/foreach}
+                                </tr>
+                            </thead>
+                            <tbody>
+                                {if $units_of_measurement == 4}<tr class="hide">{else}<tr>{/if}
+                                    {if $units_of_measurement == 2}<td class="unfocus graph" id="weight">{else}<td class="graph" id="weight">{/if}{xlt t="Weight"}</td>
+                                    {if $units_of_measurement == 2}<td class="unfocus">{else}<td>{/if}{xlt t="lbs"}</td>
+                                    {if $units_of_measurement == 2}<td class="valuesunfocus">{else}<td class='currentvalues'>{/if}
```

Comment:
Add a `p-2` for extra spacing on every td that has the `currentvalues` class or find a better way to get spacing correct here